operator/controllers/seldondeployment_explainers_test.go

```diff
@@ -156,6 +156,7 @@ var _ = Describe("Create a Seldon Deployment with explainer", func() {
 	const timeout = time.Second * 30
 	const interval = time.Second * 1
 	namespaceName := rand.String(10)
+	replicas := int32(2)
```

Comment:
Would it be worth adding also a test that checks for the explainer to be set on 0 replicas when the predictor has 0 even if explainer has 1+?

Reply:
added a test